drkafka/src/main/java/com/pinterest/doctorkafka/replicastats/ReplicaStatsManager.java

```diff
@@ -178,12 +180,30 @@ public static double get99thPercentilBytesOut(String zkUrl, TopicPartition topic
   public static Map<TopicPartition, Histogram> getTopicsBytesOutStats(String zkUrl) {
     return bytesOutStats.get(zkUrl);
   }
+  
+  public static Map<TopicPartition, Long> getProcessingStartOffsetsNew(KafkaConsumer<?, ?> kafkaConsumer,
```

Comment:
can we remove the unused `getProcessingStartOffsets` related code and reuse the method name?

Reply:
Done